examples/angular/src/app/pages/components/error/error.component.ts

```diff
@@ -35,7 +35,9 @@ import { RouterLink } from '@angular/router';
       </div>
     </section>
   </div> `,
-  styleUrl: './notFound.component.css',
+  styleUrl: './error.component.css',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class NotFoundComponent {}
+export class ErrorComponent {
+  error = input.required<PageError>();
```

Comment:
Why do we use signal input and classic inputs in some parts? I suggest to use one style in all examples.

Reply:
Fixed